packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -78,40 +78,52 @@ export const DOCUMENTATION_LINKS = [
     term: 'pci_projects_project_documentation_getting_started_term',
     description:
       'pci_projects_project_documentation_getting_started_description',
-    href:
+    asiahref:
+      'https://docs.ovh.com/asia/en/public-cloud/public-cloud-essential-information/',
+    defaulthref:
       'https://docs.ovh.com/gb/en/public-cloud/public-cloud-essential-information/',
     feature: PCI_FEATURES.LINKS.CLOUD_ESSENTIAL_INFORMATION,
   },
   {
     term: 'pci_projects_project_documentation_interface_term',
     description: 'pci_projects_project_documentation_interface_description',
-    href: 'https://docs.ovh.com/gb/en/public-cloud/public-cloud-interface/',
+    asiahref:
+      'https://docs.ovh.com/asia/en/public-cloud/public-cloud-interface/',
+    defaulthref:
+      'https://docs.ovh.com/gb/en/public-cloud/public-cloud-interface/',
```

Comment:
Why you don't made as usual ?
```suggestion
    url: {
        IN: 'https://docs.ovh.com/asia/en/public-cloud/public-cloud-interface/',
        DEFAULT: 'https://docs.ovh.com/gb/en/public-cloud/public-cloud-interface/',
    }
```